app/main.py

```diff
@@ -1,3 +1,19 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, TypeVar
+
+
+T = TypeVar("T")
```

Comment:
what is it???
remove this redundant cryptic row

Reply:
so adding "-> list[int]" is anough?